graphql/user/fields.ts

```diff
@@ -188,6 +188,11 @@ export class UserFieldsResolver {
     @Authorized(Role.ADMIN, Role.OWNER)
     async zoomZAK(@Ctx() context: GraphQLContext) {
         const { user } = context;
+
+        if (!isZoomFeatureActive()) {
+            return '';
```

Comment:
Not sure if this is a good fallback in case Zoom is disabled ...

Reply:
It's totally ok like this imho.